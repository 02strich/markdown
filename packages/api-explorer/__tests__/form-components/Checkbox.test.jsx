const React = require('react');
const { shallow } = require('enzyme');
const Checkbox = require('../../src/form-components/Checkbox');

test('should not render description', () => {
  const props = {
    label: 'test',
  };
  const checkbox = shallow(<Checkbox props={props} />);
  expect(checkbox.render().html()).toMatchSnapshot();
});
