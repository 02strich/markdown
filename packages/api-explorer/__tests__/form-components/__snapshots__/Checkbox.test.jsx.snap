// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should not render description 1`] = `"<label><input type=\\"checkbox\\"><span></span></label>"`;
